okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -480,9 +480,10 @@ int getPendingStreamSize() {
   /**
    * Finish all active streams due to an IOException, then close the transport.
    */
-  void onException(Throwable failureCause) {
-    log.log(Level.WARNING, "Transport failed", failureCause);
-    startGoAway(0, Status.UNAVAILABLE.withCause(failureCause));
+  void onException(Throwable cause) {
+    log.log(Level.WARNING, "Transport failed", cause);
+    startGoAway(0, Status.UNAVAILABLE.withCause(cause)
+        .augmentDescription(cause == null ? "No provided cause" : cause.getMessage()));
```

Comment:
Hmm, I was worried that if no message is provided, it might appear lost.  Do you care about the case where the cause is null?  This method implies that it should not be, but doesn't enforce it.

If it can be null, should I add `@Nullable` to the sig?